app/uk/gov/hmrc/customs/declarations/information/controllers/StatusController.scala

```diff
@@ -42,8 +42,14 @@ class StatusController @Inject()(val validateAndExtractHeadersAction: ValidateAn
                                 (implicit val ec: ExecutionContext) extends BackendController(cc) {
 
   def getByMrn(mrn: String): Action[AnyContent] = actionPipeline.async {
-    val searchType = Mrn(mrn)
-    implicit asr: AuthorisedRequest[AnyContent] => search(searchType)
+      val searchType = Mrn(mrn)
+      implicit asr: AuthorisedRequest[AnyContent] =>
+        if (mrn.isEmpty) {
+          logger.error("Missing mrn")
+          Future.successful(ErrorResponse(NOT_FOUND, NotFoundCode, "Invalid Search").XmlResult.withConversationId)
+        } else {
+          search(searchType)
+      }
```

Comment:
One unnecessary level of indentation. Also, should this check also be added for the other three search types (even though they are not going live yet)?

Reply:
Force pushed that change. 
I'll add them later for the other types when needed. Just now I want to test mrn in a managed env.